platform/platform_freebsd.cpp

```diff
@@ -0,0 +1,142 @@
+#include "platform/platform.hpp"
+
+#include "base/logging.hpp"
+#include "coding/file_reader.hpp"
+
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/sysctl.h>
+
+
+/// @return directory where binary resides, including slash at the end
+static bool GetBinaryFolder(string & outPath)
```

Comment:
platform_linux.cpp was written before style guide was established, and, of course, it should be rewritten in accordance with our modern practices. But I can't approve *new* code (platform_freebsd.cpp) which follows old and wrong style.